flexible-classroom/reference/platform-support.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 'Platform support'
```

Comment:
Suggestion & Reason: I think we can regard this doc as all platforms because whoever reads this doc wants to know about all the platforms Flexible Classroom support. If we divide it into four docs, it's not easy for them to check four docs separately.

Reply:
Leave as it so be consistent with platform wide logic.